Resources/Locale/en-US/deltav/store/uplink-catalog.ftl

```diff
@@ -1,2 +1,25 @@
+# Weapons
+
+# Explosives
+
+# Ammo
+
+# Utility
+uplink-reinforcement-radio-nukie-mouse-name = Mouse Operative Reinforcement Teleporter
+uplink-reinforcement-radio-nukie-mouse-desc = Calls in a specially trained mouse operative to assist you.
+
+# Implants
 uplink-bionic-syrinx-implanter-name = Bionic Syrinx Implanter
 uplink-bionic-syrinx-implanter-desc = An implant that enhances a harpy's natural talent for mimicry to let you adjust your voice to whoever you can think of.
+
```

Comment:
Same deal here

Reply:
I'm talking about everything below that, for some reason it selected the wrong lines

Reply:
I can remove it

Reply:
I can remove it